test/cases/ostree.sh

```diff
@@ -450,22 +450,58 @@ curl -O "$BOOT_LOCATION"/images/boot.iso
 sudo mv boot.iso /var/lib/libvirt/images
 LOCAL_BOOT_LOCATION="/var/lib/libvirt/images/boot.iso"
 
-# Install ostree image via anaconda.
-greenprint "Install ostree image via anaconda"
-sudo virt-install  --initrd-inject="${KS_FILE}" \
-                   --extra-args="inst.ks=file:/ks.cfg console=ttyS0,115200" \
-                   --name="${IMAGE_KEY}"\
-                   --disk path="${LIBVIRT_IMAGE_PATH}",format=qcow2 \
-                   --ram 2048 \
-                   --vcpus 2 \
-                   --network network=integration,mac=34:49:22:B0:83:30 \
-                   --os-type linux \
-                   --os-variant ${OS_VARIANT} \
-                   --location "${LOCAL_BOOT_LOCATION}" \
-                   --nographics \
-                   --noautoconsole \
-                   --wait=-1 \
-                   --noreboot
+# Workaround to fix issue https://github.com/osbuild/osbuild-composer/issues/3216
+if [[ "${ID}-${VERSION_ID}" == "fedora-37" ]]; then
+    greenprint "Install ostree image via anaconda"
+    sudo virt-install  --initrd-inject="${KS_FILE}" \
+                    --extra-args="inst.ks=file:/ks.cfg console=ttyS0,115200" \
+                    --name="${IMAGE_KEY}"\
+                    --disk path="${LIBVIRT_IMAGE_PATH}",format=qcow2 \
+                    --ram 2048 \
+                    --vcpus 2 \
+                    --network network=integration,mac=34:49:22:B0:83:30 \
+                    --os-type linux \
+                    --os-variant ${OS_VARIANT} \
+                    --location "${LOCAL_BOOT_LOCATION}",initrd=images/pxeboot/initrd.img,kernel=images/pxeboot/vmlinuz \
```

Comment:
Can we make just the `--location` argument conditional? Instead of duplicating the whole command, we can just do something like I did in the following pseudo-code:

```
if distro = f37:
    location_arg = "${LOCAL_BOOT_LOCATION}",initrd=images/pxeboot/initrd.img,kernel=images/pxeboot/vmlinuz
else:
    location_arg = "${LOCAL_BOOT_LOCATION}"

sudo virt-install ... --location $location_arg
```

I think it would be cleaner.

Reply:
Thank you Ondrej, it's more cleaner.